concent_api/middleman/tests/test_middleman.py

```diff
@@ -0,0 +1,54 @@
+import os
+import signal
+import threading
+import time
+
+from assertpy import assert_that
+import mock
+import pytest
+
+from middleman.constants import EXIT_STATUS_ADDRESS_IN_USE_ERROR
+from middleman.constants import EXIT_STATUS_OK
+from middleman.middleman_server import MiddleMan
+
+
+class TestMiddleManServer:
+    patcher = None
+    crash_logger_mock = None
+
+    def setup_method(self):
+        self.patcher = mock.patch("middleman.middleman_server.crash_logger")
+        self.crash_logger_mock = self.patcher.start()
+
+    def teardown_method(self):
+        self.patcher.stop()
+
+    def test_that_if_ctrl_c_is_pressed_application_will_exit_with_status_0(self, event_loop):
```

Comment:
I would change that to `test_that_if_keyboard_interrupt_is_raised_...`

Reply:
ok

Reply:
done